pkg/az/az.go

```diff
@@ -84,8 +84,8 @@ type AvailabilityZoneSelector struct {
 func NewSelectorWithDefaults(ec2api ec2iface.EC2API) *AvailabilityZoneSelector {
 	avoidZones := map[string]bool{
 		// well-known over-populated zones
-		"us-east-1a": true,
-		"us-east-1b": true,
+		// "us-east-1a": true,
```

Comment:
I am not quite convinced that we have to take this out completely... WDYT?

Reply:
We shouldn't need this in an ideal world and I suppose it is helpful to avoid known over populated zones.

If we automatically retried when getting `UnsupportedAvailabilityZoneException` then this is not needed.